nimh_minimal/nimh_minimal_schema

```diff
@@ -2,7 +2,7 @@
     "@context": [
         "https://raw.githubusercontent.com/ReproNim/reproschema/1.0.0-rc4/contexts/generic",
         {
-            "activity_path": "https://raw.githubusercontent.com/ReproNim/reproschema-library/80867e36fb2c00563290486bf3f3bbeb3198f5cb/activities/"
+            "activity_path": "https://raw.githubusercontent.com/ReproNim/reproschema-library/enh/rc4/a23a13875c7262c0bd0d77bd90c1ec296c6d1116/activities/"
```

Comment:
i merged this, but i think this is incorrect. the branch name should not be there.

Reply:
Oops, I will double-check next time. I created a new PR: https://github.com/ReproNim/nimh-minimal/pull/6